app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -45,7 +45,7 @@ class ProjectBoard extends React.Component {
             />
           }
         >
-          <Stories stories={this.props.chillyBinStories} />
+          <Stories stories={this.props.chillyBinStories} isChillyBin />
```

Comment:
Instead of doing prop drilling (passing this prop aaaall the way down until you need it) here, we could use this method (https://github.com/Codeminer42/cm42-central/blob/master/app/assets/javascripts/models/beta/column.js#L4-L6) passing the story only where we need it. It's safer to use this method because this is the centralize source of truth about how to say if a story is from the chilly bin column.